app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1332,8 +1332,12 @@ pub fn crumbs_overlap(src: &[span_tree::Crumb], tgt: &[span_tree::Crumb]) -> boo
 #[derive(Debug, Clone, CloneRef)]
 #[allow(missing_docs)] // FIXME[everyone] Public-facing API should be documented.
 pub struct GraphEditorModelWithNetwork {
-    pub model:   GraphEditorModel,
+    pub model: GraphEditorModel,
     pub network: frp::Network,
+    /// A network responsible for panning camera to a node newly created by the user. The network
+    /// is created when a new node is created, and dropped after the first update to the node's
+    /// position & bounding box.
+    pub network_for_camera_pan_to_new_node: Rc<RefCell<Option<frp::Network>>>,
```

Comment:
I would not make it a GraphEditorModel field, as it is used in just one place, and the network can be kept inside the clojure.